tribe-ext-formatted-event-date-shortcode/index.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+* Plugin Name: The Events Calendar Extension: Formatted Event Date Shortcode
+* Description: Output event Start Time or End Time in <a href="http://php.net/manual/en/function.date.php" target="_blank">whatever valid date/time format you want</a>. Example: [tribe_formatted_event_date id=1234 format="F jS, Y" timezone="America/New_York"] would output like "August 2nd, 2017". <strong>NOTE:</strong> "T" and other timezone output formats will be stripped from the "format" argument <em>if</em> you specify a "timezone" argument as to avoid misstatements because "T" and others will output your WordPress timezone, not taking into account this shortcode's "timezone" argument.
+* Version: 1.0.0
+* Extension Class: Tribe__Extension__Formatted_Event_Date_Shortcode
+* Author: Modern Tribe, Inc.
+* Author URI: http://m.tri.be/1971
+* License: GPLv2 or later
+* License URI: https://www.gnu.org/licenses/gpl-2.0.html
+*/
+
+// Created 2016-09-15 for https://theeventscalendar.com/support/forums/topic/_eventstartdate-format/
+// Turned into an Extension by Cliff on 2016-12-06
+
+/*
+*** Usage Examples: ***
+
+1)
+Output Event ID 1234's Start Date, like
+*** August 2, 2017 at 4:00pm ***
+(notice double escaping the word "at")
+[tribe_formatted_event_date id=1234 format="F j, Y \\a\\t g:ia"]
+
+2)
+Output Event ID 1234's End Date in a different timezone, like
+*** 02nd August, 2017 @ 18:00 PM ***
+[tribe_formatted_event_date id=1234 timezone="America/New_York" format="dS F, Y @ G:i A" start_end="end"]
+
+NOTE: Timezones (including timezone abbreviations) are not allowed in the "format" shortcode argument; they will be removed. Therefore, if the event is 11am CST but you want to display it as 12pm EST, you will need to add the "EST" manually after (i.e. outside of) the shortcode.
+
+3)
+Output current single Event's/post's Start Date like
+*** 02nd August 2017 ***
+[tribe_formatted_event_date format="dS F Y"]
+
+*/
+
+
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+   return;
+}
+
+
+
+
+/**
+* Extension main class, class begins loading on init() function.
+*/
+class Tribe__Extension__Formatted_Event_Date_Shortcode extends Tribe__Extension {
+	
+	// Define the shortcode name
+	public $shortcode_name = 'tribe_formatted_event_date';
+	
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 */
+	public function construct() {
+		// add version here in addition to the header comment if/when there is an update to this extension
+		// $this->set( 'version', '1.0.0' );
+		
+		// Each plugin required by this extension
+		$this->add_required_plugin( 'Tribe__Events__Main', '4.3.3' );
+		
+		// Set the extension's tec.com URL
+		// $this->set_url( 'https://theeventscalendar.com/extensions/formatted-event-date-shortcode/' )
```

Comment:
Do you think we could add this on tec.com ? You should see an extensions section in WP Admin and you can add it like any other article. Then you could uncomment this block of code and it will be ready to be official.